includes/gateways/paypal-standard.php

```diff
@@ -693,8 +695,9 @@ function give_get_paypal_button_type() {
  * @return string
  */
 function give_build_paypal_item_title( $payment_data ) {
```

Comment:
@mehul0810 this function will be useful outside of the PayPal gateway. Rename it to `give_payment_gateway_item_title()` and place it above https://github.com/WordImpress/Give/blob/master/includes/misc-functions.php#L395

Reply:
@DevinWalker Moved function to `misc-functions.php` above line no 395